quadrat/functions.php

```diff
@@ -37,9 +37,25 @@ function quadrat_scripts() {
  * This may not be needed once https://github.com/WordPress/gutenberg/issues/25612#issuecomment-819419024 is addressed
  */
 function quadrat_override_index_template( $template ) {
-	if ( is_home() || is_front_page() ) :
-		$template = locate_template( array( 'index.php' ) );
-	endif;
+
+	switch ($template) {
+		case is_home() || is_front_page() :
+			$template = locate_template( array( 'index.php' ) );
+			break;
```

Comment:
Rather than setting a variable and breaking can this just be an early exit?

return locate_template( array( 'index.php' ) );

Reply:
Also, can the default case just be `index.php` or do we need to have a case for `is_home() || is_front_page()`?